src/interfaces/podcast.ts

```diff
@@ -0,0 +1,10 @@
+export interface Podcast {
+    mediaType: string;
+    title: string;
+    publishDate: string;
+    description?: string;
+    rawDescription?: string;
+    slug?: string;
+    url: string;
+    thumbnailUrl: string;
```

Comment:
A nitpick, but I would like for us to alphabetize these please for the sake of better organization.

Reply:
sure :)